examples/basic/layouts/css_layouts.py

```diff
@@ -1,6 +1,5 @@
-from bokeh.io import save
 from bokeh.models.dom import Div, Styles
-from bokeh.plotting import figure
+from bokeh.plotting import figure, save
```

Comment:
In this example `show()` fails with an error

```
ValueError: Invalid object to show. The object to passed to show must be one of:

* a UIElement (e.g. a plot, figure, widget or layout)
* a Bokeh Application
* a callable suitable to an application FunctionHandler
```

and so I had to revert the changes.

Reply:
This is unfortunate, `save()` and `show()` should support the same kinds of objects, at least in the context of standalone output. This deserves its own issue though.